pkg/apis/network.harvesterhci.io/v1beta1/nodenetwork.go

```diff
@@ -33,7 +33,7 @@ type NodeNetworkSpec struct {
 	Type NetworkType `json:"type,omitempty"`
 
 	// +optional
-	NIC string `json:"nic,omitempty"`
+	NetworkInterface string `json:"nic,omitempty"`
```

Comment:
Generally, we should keep the json key and the field name consistent. If we don't change the json key, we should keep the filed name unchanged

Reply:
It's temporary. We consider the compatibility between single cluster UI and Rancher UI. Rancher UI can not be modified now while single cluster UI will add this new feature. https://github.com/harvester/harvester/issues/1273